### DISCUSSION TITLE
Pyg without pyg-lib

### DISCUSSION THREAD

**Discussion by denadai2** [pyg-team/pytorch_geometric#7455] [General]
Dear developers,
why are we firing this message https://github.com/pyg-team/pytorch_geometric/blob/f0c9fed058038a3288185e1f583069e6d98a28b4/torch_geometric/sampler/neighbor_sampler.py#L57 if pyg-lib does not support non-directed graphs? Isn't it too early?

**Comment by rusty1s** [comment]
That's a good point. Let me remove this warning in case `directed=False`.

**Comment by rusty1s** [comment]
I fixed this in https://github.com/pyg-team/pytorch_geometric/pull/7468.
